src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/avoid.cpp

```diff
@@ -50,9 +50,9 @@ RMSNormPattern::RMSNormPattern(const std::shared_ptr<ov::npuw::online::Snapshot>
     // Note: Use [=] to make sure the above objects stay alive in the callback
     auto callback = [=](ov::pass::pattern::Matcher& m) {
         auto& node_to_output = m.get_pattern_value_map();
-        auto matched_power = node_to_output.at(power).get_node_shared_ptr();
-        auto matched_reduce = node_to_output.at(reduce).get_node_shared_ptr();
-        auto matched_add = node_to_output.at(add).get_node_shared_ptr();
+        auto matched_power = node_to_output.at(std::move(power)).get_node_shared_ptr();
+        auto matched_reduce = node_to_output.at(std::move(reduce)).get_node_shared_ptr();
+        auto matched_add = node_to_output.at(std::move(add)).get_node_shared_ptr();
```

Comment:
same here. Why on earth we pass ownership to `.at` if all we need to do is to pass a key?

Reply:
I think, that just because we don't need this variable anywhere else after the call, we just can transfer ownership if it is beneficial for preformance - that was the Coverity logic